### PR TITLE
871811 - Added brush setting content for tree map control.

### PR DESCRIPTION
**Pull request by Karthickmani97**
### Description

Added brush setting content for tree map control.

### CODE REVIEW

File: MAUI/TreeMap/brush-settings.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
+---
+layout: post
+title: Brush settings in .NET MAUI TreeMap (SfTreeMap) control | Syncfusion
```
Comment (M-A32) [inline MAUI/TreeMap/brush-settings.md:3]: title: Brush Settings in .NET MAUI TreeMap (SfTreeMap) Control | Syncfusion

File: MAUI/TreeMap/brush-settings.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+description: Learn here how to apply brush settings for tree map items in Syncfusion .NET MAUI TreeMap control, enhancing the tree map visualization.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Leaf item brush settings in .NET MAUI TreeMap (SfTreeMap)
+
+The brush settings is used to customize the fill colors for leaf items based on ranges or values, offering four brush settings: `UniformBrushSettings,` `RangeBrushSettings,` `DesaturationBrushSettings,` and `PaletteBrushSettings.` Each setting provides unique options for defining and applying color schemes, enhancing the visualization of the tree map.
```
Comment (M-A32) [inline MAUI/TreeMap/brush-settings.md:12]: The brush settings are used to customize the fill colors for leaf items based on ranges or values, offering four brush settings: `UniformBrushSettings,` `RangeBrushSettings,` `DesaturationBrushSettings,` and `PaletteBrushSettings.` Each setting provides unique options for defining and applying color schemes, enhancing the visualization of the treemap.

File: MAUI/TreeMap/brush-settings.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+
+## Uniform brush settings
+
+The uniform brush settings in `SfTreeMap` assigns a single color uniformly for leaf items representing data. Customize the background brush for tree map leaf items using the `Brush` property of `TreeMapUniformBrushSettings`.
```
Comment (M-A32) [inline MAUI/TreeMap/brush-settings.md:16]: The uniform brush settings in `SfTreeMap` assign a single color uniformly for leaf items representing data. Customize the background brush for treemap leaf items using the `Brush` property of `TreeMapUniformBrushSettings`.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/TreeMap/brush-settings.md (hunk 1 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/brush-settings.md]: Updated.

File: MAUI/TreeMap/brush-settings.md (hunk 2 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/brush-settings.md]: Updated.

File: MAUI/TreeMap/brush-settings.md (hunk 3 above)
Comment (Karthickmani97) [reply MAUI/TreeMap/brush-settings.md]: Updated.

**Review by MuniappanSubramanian — APPROVED** [review]

File: MAUI/TreeMap/brush-settings.md [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The `TreeMapRangeBrushSettings` is applicable only when both the `RangeColorValuePath` property of `SfTreeMap` and the `ShowLegend` property of `LegendSettings` are enabled.
+
+## Desaturation brush settings
+
+The desaturation brush settings in the `SfTreeMap` modify color saturation based on data values, enabling customization of the background brush and desaturation range.
```
Comment (M-A32) [inline MAUI/TreeMap/brush-settings.md:235]: The desaturation brush settings in the `SfTreeMap` modify color saturation based on the data values, enabling customization of the background brush and desaturation range.

**Review by M-A32 — COMMENTED** [review]
I have completed the review.

**Pull request merged**